Composer/packages/lib/ui-shared/src/components/ProvisionHandoff/ProvisionHandoff.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import { Dialog, DialogType, DialogFooter } from 'office-ui-fabric-react/lib/Dialog';
+import { PrimaryButton } from 'office-ui-fabric-react/lib/Button';
+import formatMessage from 'format-message';
+
+import { instructionStyles, copyPasteStyles, dialog } from './styles';
+type ProvisionHandoffProps = {
+  title: string;
+  developerInstructions: string;
+  handoffInstructions: string;
+  hidden: boolean;
+  onDismiss: () => void;
+};
+
+export const ProvisionHandoff = (props: ProvisionHandoffProps) => {
+  const closeDialog = () => {
```

Comment:
Nit: Unless you are saving this for some logic, you can make onClick call onDismiss directly.

Reply:
there might be logic will reserve judgement on this til its 100% done with ui

Reply:
done